<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>design</class>
 <widget class="QMainWindow" name="design">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>780</width>
          <height>386</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QLabel" name="imageLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>782</width>
           <height>57</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>对比度</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="alphaSlider">
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="pageStep">
           <number>28</number>
          </property>
          <property name="sliderPosition">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="TextLabel">
          <property name="text">
           <string>80</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QSlider" name="betaSlider">
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="pageStep">
           <number>28</number>
          </property>
          <property name="sliderPosition">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="TextLabel_2">
          <property name="text">
           <string>80</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string> 亮度 </string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuAddLittleIcon">
     <property name="title">
      <string>addLittleIcon</string>
     </property>
     <addaction name="actionIconAdd"/>
     <addaction name="actionIconCancel"/>
     <addaction name="actionIconSave"/>
    </widget>
    <widget class="QMenu" name="menuAddBoard">
     <property name="title">
      <string>addBoard</string>
     </property>
     <addaction name="actionAddBoard"/>
     <addaction name="actionSave_2"/>
    </widget>
    <addaction name="actionDetectface"/>
    <addaction name="actionAddNoise"/>
    <addaction name="actionDeNoise"/>
    <addaction name="actionEraser"/>
    <addaction name="actionWhiteBalance"/>
    <addaction name="menuAddLittleIcon"/>
    <addaction name="menuAddBoard"/>
    <addaction name="actionBeauty"/>
    <addaction name="actionCutout"/>
    <addaction name="actionDetectEdge"/>
    <addaction name="actionRedo"/>
    <addaction name="actionImageMatch"/>
    <addaction name="actionFudiao"/>
    <addaction name="actionHuaijiu"/>
    <addaction name="actionSketch"/>
    <addaction name="actionCartoon"/>
   </widget>
   <widget class="QMenu" name="menuAdvance">
    <property name="title">
     <string>Advance</string>
    </property>
    <addaction name="actionStitch"/>
    <addaction name="actionFaceTracking"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuAdvance"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDetectface"/>
   <addaction name="actionAddNoise"/>
   <addaction name="actionDeNoise"/>
   <addaction name="actionEraser"/>
   <addaction name="actionWhiteBalance"/>
   <addaction name="actionSave_2"/>
   <addaction name="actionEqualization"/>
   <addaction name="actionBeauty"/>
   <addaction name="actionCutout"/>
   <addaction name="actionDetectEdge"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="imageprocessor.qrc">
     <normaloff>:/open/images/open.png</normaloff>
     <normalon>images/open.png</normalon>:/open/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imageprocessor.qrc">
     <normaloff>:/save/images/save.png</normaloff>:/save/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDetectface">
   <property name="text">
    <string>detectface</string>
   </property>
  </action>
  <action name="actionAddNoise">
   <property name="text">
    <string>addNoise</string>
   </property>
  </action>
  <action name="actionDeNoise">
   <property name="text">
    <string>deNoise</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>eraser</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionWhiteBalance">
   <property name="text">
    <string>whiteBalance</string>
   </property>
  </action>
  <action name="actionIconSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionIconAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAddBoard">
   <property name="text">
    <string>addBoard</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionEqualization">
   <property name="text">
    <string>equalization</string>
   </property>
  </action>
  <action name="actionBeauty">
   <property name="text">
    <string>Beauty</string>
   </property>
  </action>
  <action name="actionCutout">
   <property name="text">
    <string>cutout</string>
   </property>
  </action>
  <action name="actionDetectEdge">
   <property name="text">
    <string>DetectEdge</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionIconCancel">
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionStitch">
   <property name="text">
    <string>stitch</string>
   </property>
  </action>
  <action name="actionImageMatch">
   <property name="text">
    <string>imageMatch</string>
   </property>
  </action>
  <action name="actionFaceTracking">
   <property name="text">
    <string>FaceTracking</string>
   </property>
  </action>
  <action name="actionFudiao">
   <property name="text">
    <string>浮雕</string>
   </property>
  </action>
  <action name="actionHuaijiu">
   <property name="text">
    <string>怀旧</string>
   </property>
  </action>
  <action name="actionSketch">
   <property name="text">
    <string>素描</string>
   </property>
  </action>
  <action name="actionCartoon">
   <property name="text">
    <string>油画</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imageprocessor.qrc"/>
 </resources>
 <connections/>
</ui>
